website/content/docs/ecs/terraform/install.mdx

```diff
@@ -7,34 +7,42 @@ description: >-
 
 # Installation with Terraform
 
-This topic describes how to use the [`mesh-task`](https://registry.terraform.io/modules/hashicorp/consul-ecs/aws/latest/submodules/mesh-task) Terraform module to launch your application in AWS ECS as part of Consul service mesh. If you do not use Terraform, see the [Manual Installation](/docs/ecs/manual-installation) page to install Consul on ECS without Terraform.
+This topic describes how to use HashiCorp’s Terraform modules to launch your application in AWS ECS as part of Consul service mesh. If you do not use Terraform, refer to the [Manual Installation](/docs/ecs/manual-installation) page to install Consul on ECS without Terraform.
 
-This topic does not include instructions for creating all AWS resources necessary to install Consul, such as a VPC or the ECS cluster. Refer to the linked guides in the [Getting Started](/docs/ecs#getting-started) section for complete, runnable examples.
+This topic does not include instructions for creating all AWS resources necessary to install Consul, such as a VPC or the ECS cluster. Refer to the guides in the [Getting Started](/docs/ecs#getting-started) section for complete and runnable examples.
 
 ## Overview
 
-This topic describes the following procedure:
+The following procedure describes the general workflow:
 
 1. Create Terraform configuration files for the necessary components:
 
-    * [ECS task definition](#using-the-mesh-task-module): Use the `mesh-task` module to create an ECS task definition for Consul on ECS
-    * [ECS service](#ecs-service): Use the `aws_ecs_service` resource to create an ECS service that schedules service mesh tasks to run on ECS
+    - [ECS task definition](#create-the-task-definition): Use the HashiCorp Terraform modules to create the ECS task definition.
+    - [ECS service](#ecs-service): Use the `aws_ecs_service` resource to create an ECS service that schedules service mesh tasks to run on ECS.
 
 2. [Run Terraform](#running-terraform) to deploy the resources in AWS
 
-## Prerequisites
+If you want to operate Consul with ACLs enabled (recommended), follow the instructions in the [Secure Configuration](/docs/ecs/terraform/secure-configuration) documentation. ACLs provide network security for production-grade deployments.
+
+## Requirements
 
 * You should have some familiarity with using Terraform. Refer to the [Terraform documentation](https://www.terraform.io/docs) to learn about infrastructure as code and how to get started with Terraform.
 * You should also be familiar with AWS ECS before following these instructions. See [What is Amazon Elastic Container Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/Welcome.html) for details.
+* If you intend to [use the `gateway-task` module to deploy mesh gateways](#configure-the-gateway-task-module), all Consul server and client agents in all datacenters must have TLS and gossip encryption enabled (refer to [Encryption](/docs/security/encryption) for instructions). 
```

Comment:
> The "Encryption" link provided here does not really instruct the user on how to enable encryption for the ECS deployment. It is more informative about general Consul encryption. I suggest either pointing to the /docs/ecs/terraform/secure-configuration section or changing "instructions" to "more information"

@cthain  Ah, OK. Just to be clear, the instructions for enabling encryption described in this section? /docs/ecs/terraform/secure-configuration#deploy-services - If so, we should update the headings on that page to more accurately reflect the steps; maybe even get rid of the headings because the procedure seems relatively short.

Reply:
> Just to be clear, the instructions for enabling encryption described in this section? /docs/ecs/terraform/secure-configuration#deploy-services

Yes, that's correct. I am fine with your suggestion to reorg that section if you think it will make it more clear.